generators/app/tasks/installDependencies.js

```diff
@@ -11,7 +11,8 @@ const DEPENDENCIES = [
   'redux-thunk',
   'reselect',
   'seamless-immutable',
-  'react-native-dotenv'
+  'react-native-dotenv',
+  'react-native-huawei-protected-apps'
```

Comment:
we should only install this dependency if push notifications feature is enabled. check this out https://github.com/Wolox/wolmo-bootstrap-react-native/pull/15/files#diff-237b39754a118903c976a389eb22a016R50